### PR TITLE
Sub grade walls handling

### PR DESCRIPTION
**Pull request by TfedUD**
Temp sub-grade handling implemented, does not reflect location being sub-grade in *.inp yet, temp fix to reduce errors to continue ID'ing correctly, and addressing other issues

### CODE REVIEW

**Comment by TfedUD** [comment]
#36

**Comment by TfedUD** [comment]
feats and changes TLDR:
1.  sub-grade walls are not ignored on import, though more is needed to correctly reflect the BC of sub-grade in   eQuest. #36 
2. eQuest shade naming convention modified to handle big long hashed names/ID's they get some times when not explicitly named and there are alot of them. 
3. appended applicable construction sets from ```generic_construction_set``` to the ```df_envelope_constrs``` so generic cons sets now function =>  doe2. #34 
4. Added always solve intersection/adjacency at the story level:
    Passes the "made the lab work" check, that initially ID'd the need for it. # #35

**Comment by TfedUD** [comment]
Implemented short_name(name,number) to window naming convention
created placeholder systems for each story, serving all story.room_2ds as zones

File: dragonfly_doe2/doe/shades.py [hunk 1]
```diff
@@ -99,10 +99,11 @@ def from_df_context_shades(cls, df_shades: [ContextShade]):
         """Generate doe2 fixed shades from dragonfly context shades"""
         shade_faces = []
 
-        for shade in df_shades:
+        for i, shade in enumerate(df_shades):
+            shade_i = i
```
Comment (mostaphaRoudsari) [inline dragonfly_doe2/doe/shades.py:103]:
```suggestion
        for shade_i, shade in enumerate(df_shades):
```

**Review by mostaphaRoudsari — COMMENTED** [review]
Thanks @TfedUD! Looks great. See my comments.

File: dragonfly_doe2/doe/hvac.py [hunk 2]
```diff
@@ -1,26 +1,93 @@
...
+    Args:
+        name: room display name
+        hx_setpoint: heating setpoint temperature (*F)
+        cx_setpoint: cooling setpoint temperature (*F)
+    Init method(s):
+        1. from_room(room: DFRoom)-> doe_zone:
+        """
+    name: str
+    hx_setpoint: float
+    cx_setpoint: float
```
[the remark this answers is not in the document]
Comment (TfedUD) [reply dragonfly_doe2/doe/hvac.py:26]: habit I picked up... 🤷🏼

File: dragonfly_doe2/doe/shades.py (hunk 1 above)
Comment (TfedUD) [reply dragonfly_doe2/doe/shades.py:103]: ew.. thank you lol

**Pull request merged**